packages/vue/src/components/molecules/SfRange/SfRange.vue

```diff
@@ -0,0 +1,231 @@
+    },
+    /*
+     * Horizontal orientation if true, vertical orientation if false
+     */
+    horizontalOrientation: {
+      type: Boolean,
+      default: true,
+    },
+    /*
+     * Ltr orientation if true, rtl orientation if false
+     */
+    ltrDirection: {
+      type: Boolean,
+      default: true,
+    },
+    /*
+     * Boolean for showing the tooltips
+     */
+    tooltips: {
+      type: Boolean,
+      default: false,
+    },
+    /*
+     * Config for toolips formatting (from wNumb library):
+     * decimals - number of digits after dot/comma
+     * mark - sign separating the fraction value from integer
+     * thousand - sign separating the thousands from hundreds
+     * prefix - sign before value
+     * suffix - sign after the value
+     * negative - sign for negative values, by default it is set to "-"
+     */
+    formatTooltipsValues: {
+      type: Object,
+      default: () => {
+        return {
+          decimals: 2,
+          mark: ".",
+          thousand: " ",
+          prefix: "$",
+          suffix: "",
+          negative: "",
+        };
+      },
+    },
+  },
+  data() {
+    return {
+      config: {},
+    };
+  },
+  watch: {
+    min: {
+      handler(value) {
+        if (this.$refs.range) {
+          return this.updateRangeOptions({
+            range: {
+              min: value >= this.max ? this.max - 1 : value,
+              max: this.max,
+            },
+          });
+        }
+      },
+      immediate: true,
+    },
+    max: {
+      handler(value) {
+        return this.updateRangeOptions({
+          range: {
+            min: this.min,
+            max: value <= this.min ? this.min + 1 : value,
+          },
+        });
+      },
+      immediate: true,
+    },
+    step: {
+      handler(value) {
+        return this.updateRangeOptions({
+          step: value,
+        });
+      },
+      immediate: true,
+    },
+    tooltips: {
+      handler(value) {
+        value
+          ? (value =
+              this.slidersInitialValues.length === 2
+                ? [
+                    wNumb(this.formatTooltipsValues),
+                    wNumb(this.formatTooltipsValues),
+                  ]
+                : [wNumb(this.formatTooltipsValues)])
+          : value;
+        return this.updateRangeOptions({
+          tooltips: value,
+        });
+      },
+      immediate: true,
+    },
+    formatTooltipsValues: {
+      handler(value) {
+        if (this.tooltips)
+          return this.updateRangeOptions({
+            tooltips:
+              this.slidersInitialValues.length === 2
+                ? [
+                    wNumb(this.formatTooltipsValues),
+                    wNumb(this.formatTooltipsValues),
+                  ]
+                : [wNumb(this.formatTooltipsValues)],
+          });
+      },
+      immediate: true,
+    },
+    horizontalOrientation: {
+      handler(value) {
+        value ? (value = "horizontal") : (value = "vertical");
```

Comment:
Do you need it? Couldn't we below just use:
```js
orientation: value ? 'horizontal' : 'vertical',
```

Reply:
it changes boolean prop into value = horizontal or vertical, maybe I change the value into orientationValue to be clearer: 
```
handler(value) {
				let orientationValue = "horizontal";
        value ? orientationValue = "horizontal" : orientationValue = "vertical";
        return this.resetAndChangeOption({
          orientation: orientationValue,
        });
      },
```